src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -48,14 +49,36 @@ void test1() throws Exception {
 		dataModel.latestState.put(deviceA, createState());
 		dataModel.latestState.put(deviceB, createState());
 
-		final int txPower = 16;
+		// Test providing tx power
+		for (int txPower: Arrays.asList(-1, 16, 27)) {
+			TPC optimizer = new RandomTxPowerInitializer(
+					dataModel, TEST_ZONE, deviceDataManager, txPower
```

Comment:
style: always use 1 tab to indent, never 2 tabs

Reply:
The default of my Intellij use 2 tabs for "continuation indent". Will change it.